ivy/functional/frontends/torch/tensor.py

```diff
@@ -683,6 +683,15 @@ def clamp_(self, min=None, max=None, *, out=None):
         self._ivy_array = self.clamp(min=min, max=max, out=out).ivy_array
         return self
 
+    @with_unsupported_dtypes({"1.11.0 and below": ("float16")}, "torch")
+    def clip(self, min=None, max=None):
+        return torch_frontend.clip(self._ivy_array, min=min, max=max)
+
+    @with_unsupported_dtypes({"1.11.0 and below": ("float16")}, "torch")
+    def clip_(self, min=None, max=None):
+        self._ivy_array = self.clip(min=min, max=max).ivy_array
+        return self
+
```

Comment:
Since `clip` is also an alias for `clamp`, and `clip_` is an alias for `clamp_`, can you kindly add them [here](https://github.com/unifyai/ivy/blob/a86a2fb82dfcf45870a178d723b0df2eca728b75/ivy/functional/frontends/torch/tensor.py#L891-L893) as well in this file?